client/ayon_speedtree/api/pipeline.py

```diff
@@ -0,0 +1,326 @@
+
+
+def get_load_context_metadata():
+    """Get the context data from the related json file
+    ("context.json") which stores in .sptree_metadata/context
+    folder in the project work directory.
+
+    The context data includes the project name, folder path and
+    task name.
+
+    Returns:
+        list: context data
+    """
+    file_content = {}
+    work_dir = get_workdir()
+    json_dir = os.path.join(
+        work_dir, ".sptree_metadata", SPTREE_SECTION_NAME_CONTEXT).replace(
+            "\\", "/"
+        )
+    if not os.path.exists(json_dir):
+        return file_content
+    file_list = os.listdir(json_dir)
+    if not file_list:
+        return file_content
+    for file in file_list:
+        with open (f"{json_dir}/{file}", "r") as data:
+            content = ast.literal_eval(str(data.read().strip()))
+            file_content.update(content)
+            data.close()
+    return file_content
+
+
+
+def set_current_file(filepath=None):
+    """Function to store current workfile path
+
+    Args:
+        filepath (str, optional): current workfile path. Defaults to None.
+    """
+    work_dir = get_workdir()
+    txt_dir = os.path.join(
+        work_dir, ".sptree_metadata").replace(
+            "\\", "/"
+    )
+    os.makedirs(txt_dir, exist_ok=True)
+    txt_file = f"{txt_dir}/current_file.txt"
+    if filepath is None:
+        with open(txt_file, "w"):
+            pass
+        return filepath
+    filepath_check = tmp_current_file_check()
+    if filepath_check.endswith("spm"):
+        filepath = os.path.join(
+            os.path.dirname(filepath), filepath_check).replace("\\", "/")
+    with open (txt_file, "w") as current_file:
+        current_file.write(filepath)
+        current_file.close()
+
+
+def tmp_current_file_check():
+    """Function to find the latest .spm file used
+    by the user in Speedtree.
+
+    Returns:
+        file_content (str): the filepath in .spm format.
+            If the filepath does not end with '.spm' format,
+            it returns None.
+    """
+    output_file = tempfile.NamedTemporaryFile(
+        mode="w", prefix="a_sptree_cfc", suffix=".txt", delete=False
+    )
+    output_filepath = output_file.name.replace("\\", "/")
+    output_file.write(output_filepath)
+    output_file.close()
+    with open(output_filepath) as data:
+        file_content = str(data.read().strip()).rstrip('\x00')
+    os.remove(output_filepath)
+    return file_content
+
+
+def show_tools_dialog():
+    """Show dialog with tools.
+
+    Dialog will stay visible.
+    """
+    from ayon_speedtree.api import tools_ui
+
+    tools_ui.show_tools_dialog()
+
+
+def show_creator():
+    host_tools.show_creator()
+
+
+def show_loader():
+    host_tools.show_loader(use_context=True)
+
+
+def show_publisher():
+    host_tools.show_publish()
+
+
+def show_manager():
+    host_tools.show_scene_inventory()
+
+
+def show_workfiles():
+    host_tools.show_workfiles(use_context=True)
```

Comment:
These wrappers seem very thin and redundant, feel free to remove and just use `host_tools` directly in other areas?